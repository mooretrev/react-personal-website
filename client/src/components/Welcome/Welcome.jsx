import React from 'react';
import Card from '@material-ui/core/Card';
import CardContent from '@material-ui/core/CardContent';
import CardMedia from '@material-ui/core/CardMedia';
import Typography from '@material-ui/core/Typography';
import useStyles from '../../styles/Index.jsx';
import RisheilPic from '../../static/images/rish.png';

/* eslint-disable max-len */

export default function Welcome() {
  const classes = useStyles();

  return (
    <Card className={classes.card}>
      <CardMedia
        component="img"
        alt="Risheil and me"
        src={RisheilPic}
        title="Risheil and Me"
      />
      <CardContent>
        <Typography id="header" gutterBottom variant="h5" component="h2">
          About Me
        </Typography>
        <Typography variant="body2" color="textSecondary" component="p">
          Howdy, I am glad you are here. My name is Trevor Moore. I am currently studying at Texas A&M class of 2021. A-Whoop! I am currently looking for an internship for Summer 2021. My passions include powerlifting and water skiing. In the powerlift tab, you can see the different power lifting meets we had and the totals we achieved. Along with the current program I am doing and my current maxes.
        </Typography>
      </CardContent>
    </Card>
  );
}
