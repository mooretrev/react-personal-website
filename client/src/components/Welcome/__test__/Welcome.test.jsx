import React from 'react';
import { shallow } from 'enzyme';
import Typography from '@material-ui/core/Typography';
import toJson from 'enzyme-to-json';
import Welcome from '../Welcome.jsx';

describe('<Welcome />', () => {
  it('should render correctly', () => {
    const wrapper = shallow(<Welcome />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });

  it('should have the about me header', () => {
    const wrapper = shallow(<Welcome />);
    const header = (
      <Typography id="header" gutterBottom variant="h5" component="h2">
        About Me
      </Typography>
    );
    expect(wrapper.contains(header)).toBe(true);
  });
});
