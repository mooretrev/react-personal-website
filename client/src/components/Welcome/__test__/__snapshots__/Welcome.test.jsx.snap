// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Welcome /> should render correctly 1`] = `
<WithStyles(ForwardRef(Card))
  className="makeStyles-card-10"
>
  <WithStyles(ForwardRef(CardMedia))
    alt="Risheil and me"
    component="img"
    src="test-file-stub"
    title="Risheil and Me"
  />
  <WithStyles(ForwardRef(CardContent))>
    <WithStyles(ForwardRef(Typography))
      component="h2"
      gutterBottom={true}
      id="header"
      variant="h5"
    >
      About Me
    </WithStyles(ForwardRef(Typography))>
    <WithStyles(ForwardRef(Typography))
      color="textSecondary"
      component="p"
      variant="body2"
    >
      Howdy, I am glad you are here. My name is Trevor Moore. I am currently studying at Texas A&M class of 2021. A-Whoop! I am currently looking for an internship for Summer 2021. My passions include powerlifting and water skiing. In the powerlift tab, you can see the different power lifting meets we had and the totals we achieved. Along with the current program I am doing and my current maxes.
    </WithStyles(ForwardRef(Typography))>
  </WithStyles(ForwardRef(CardContent))>
</WithStyles(ForwardRef(Card))>
`;
